test/unit/core_test.rb

```diff
@@ -8,25 +8,6 @@ class Superstore::CoreTest < Superstore::TestCase
     assert !issue.destroyed?
   end
 
-  test 'initialize yields self' do
-    issue = Issue.new { |i| i.description = 'bar' }
-    assert_equal 'bar', issue.description
-  end
-
-  test 'dup' do
-    issue = Issue.create description: 'foo', comments: {"foo" => "bar"}
-
-    dup_issue = issue.dup
-
-    assert dup_issue.new_record?
-    refute_equal issue.id, dup_issue.id
-    assert_nil dup_issue.created_at
-    assert_nil dup_issue.updated_at
-    assert_equal issue.description, dup_issue.description
-    refute_equal issue.description.object_id, dup_issue.description.object_id
-    refute_equal issue.comments['foo'].object_id, dup_issue.comments['foo'].object_id
```

Comment:
No, this line of code would not pass. It is unfortunately the behavior of active record to not deep dup the values.

Superstore no longer overrides `initialize_dup`. I could keep the test here but it would be testing active record.